apps/video/js/video.js

```diff
@@ -174,14 +174,15 @@ if (acm) {
 navigator.mozSetMessageHandler('activity', handleActivityEvents);
 
 if (!isPhone) {
-  navigator.mozL10n.ready(function localizeThumbnailListTitle() {
-    // reload the thumbnail list title field for tablet which is the app name.
-    var req = navigator.mozApps.getSelf();
-    req.onsuccess = function() {
-      var manifest = new ManifestHelper(req.result.manifest);
-      dom.thumbnailListTitle.textContent = manifest.name;
-    };
-  });
+  // reload the thumbnail list title field for tablet which is the app name.
+  var req = navigator.mozApps.getSelf();
+  req.onsuccess = function() {
+    if (!req.result) {
+      return;
+    }
+    var manifest = new ManifestHelper(req.result.manifest);
+    dom.thumbnailListTitle.textContent = manifest.name;
+  };
```

Comment:
With l10n, does ready() run only once, or does it get called again when the locale changes? If the latter then this new code does not behave the same.

But this !isPhone code is for the old flatflish tablet and is not supported anymore anyway, so we could also just delete this. It seems absurd to look up the app name like this anyway rather than just using something in the regular locale file anyway.